Index: pagelayout.tpl
===================================================================
--- pagelayout.tpl	(revision 17414)
+++ pagelayout.tpl	(working copy)
@@ -94,12 +94,10 @@
 	        <li{if $row.columns[1]|downcase()|eq($access_type.name)} class="current_siteaccess"{/if}>
 	        {if is_set($DesignKeys:used.url_alias)}
 	            <a href={concat( "http://", $site_url,
-	                     $site_access,
-	                     $DesignKeys:used.url_alias
+                         $DesignKeys:used.url_alias
 	                     )}>{$language}</a>
 	        {else}
 	            <a href={concat( "http://", $site_url,
-	                     $site_access,
 	                     $uri_string
                          )}>{$language}</a>
             {/if}
@@ -162,7 +160,7 @@
   </div>
   <!-- Header area: END -->
 
-  
+
   <!-- Top menu area: START -->
   <div id="topmenu" class="float-break">
     {include uri='design:menu/flat_top.tpl'}
@@ -181,7 +179,7 @@
   <!-- Path area: END -->
   {/if}
 
-  
+
   <!-- Toolbar area: START -->
   <div id="toolbar">
   {if and( $current_node_id, and( is_set( $module_result.content_info.viewmode ), ne( $module_result.content_info.viewmode, 'sitemap' ) ) ) }
@@ -190,7 +188,7 @@
   </div>
   <!-- Toolbar area: END -->
 
-  
+
   <!-- Columns area: START -->
   <div id="columns" class="float-break">
     <!-- Side menu area: START -->
@@ -203,7 +201,7 @@
        </div>
     </div>
     <!-- Side menu area: END -->
-    
+
     <!-- Extra area: START -->
     <div id="extrainfo-position">
       <div id="extrainfo">
@@ -229,14 +227,14 @@
     <!-- Main area: END -->
   </div>
   <!-- Columns area: END -->
-  
+
 {cache-block keys=$access_type.name}
   {if is_unset($pagedesign)}
    {def $pagedesign = fetch( 'content', 'object', hash( 'object_id', '54' ) )}
   {/if}
-  
-{include uri='design:page_footer.tpl'} 
 
+{include uri='design:page_footer.tpl'}
+
 </div>
 <!-- Complete page area: END -->
 
